movieland-domain/src/main/java/com/gromoks/movieland/entity/ReportMovie.java

```diff
@@ -0,0 +1,67 @@
+package com.gromoks.movieland.entity;
+
+public class ReportMovie {
+    private int movieId;
+    private String title;
+    private String description;
+    private String genres;
+    private double price;
+    private double rating;
+    private int reviewCount;
```

Comment:
Date for add and edit missed from the report

Reply:
I thought it's more related to second type report that was not implemented